Frontend/ui-common/src/Components/Banner/View.purs

```diff
@@ -32,63 +34,53 @@ view push config =
     [ height WRAP_CONTENT
     , width MATCH_PARENT
     , cornerRadius 12.0
-    , margin $ MarginTop 12
     , background config.backgroundColor
     , visibility if config.isBanner then VISIBLE else GONE
     , onClick push (const OnClick)
     ]
     [  linearLayout
-        [ height WRAP_CONTENT
+        [ height MATCH_PARENT
         , weight 1.0
-        , padding $ Padding 20 7 0 0
+        , padding if (os == "IOS") then Padding 20 16 0 18 else Padding 20 13 0 18
```

Comment:
@prakharritik can you add this check only for top padding parameter.

Reply:
Done